documentation/src/main/markdown/integrations/jenkinsplugin/usersandroles.md

```diff
@@ -7,7 +7,7 @@ First you must configure a user/API token in [blackduck_product_name] so that th
 4. Click **Create New Token**. The Create New Token dialog box appears
 5. Type your name in the **Name** field.
 6. Optional: in the **Description** field, you can type a description or definition.
-7. Select **Read Access** and/or **Write Access**.
+7. Select **Read Access** and **Write Access**.
```

Comment:
We should write Read and Write Access instead of Read Access and Write Access
We have single radio button in UI with text Read and Write Access.

Reply:
Absolutely.